packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -9,6 +9,9 @@ import { Previewer } from './Previewer';
 import { UploadDropZone } from './UploadDropZone';
 import { Tracker } from './Tracker';
 
+const isUploadTask = (value: unknown): value is UploadTask =>
+  typeof (value as UploadTask)?.resume === 'function';
```

Comment:
If we have to do `value as UploadTask`, why not define it at the beginning? 

```suggestion
const isUploadTask = (value: UploadTask): value is UploadTask =>
  typeof (value)?.resume === 'function';
```

Reply:
I believe it's a standard practice to have `value` defined loosely in typeguards: https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards.

This typeguard, for example, asserts whether this loosely typed `value` is actually of type `UploadTask`, and narrows the type to it if so.